subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/SpringBootPluginSmokeTest.groovy

```diff
@@ -72,14 +73,27 @@ class SpringBootPluginSmokeTest extends AbstractPluginValidatingSmokeTest implem
         """
 
         when:
-        def buildResult = runner('assembleBootDist', 'check').build()
+        def smokeTestRunner = runner('assembleBootDist', 'check')
+        // verified manually: the 3.0.2 version of Spring Boot plugin removed the deprecated API usage
```

Comment:
We currently smoke test 2.x. Could you please open an issue about smoke testing boot 3.x instead.

Reply:
Here it is: https://github.com/gradle/gradle/issues/23937

Reply:
Thanks